### PR TITLE
Added CardBin to payload

### PR DESCRIPTION
**Pull request by RamananMobilepay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -195,6 +195,7 @@ A Visa Token Service (VTS) response like this: <br />
 	"cardType": "string",	
 	"cardIssuedInCountryCode": "string",
 	"maskedCardNumber":"string",
+        "cardBin":"string",
```
Comment (ct-clearhaus) [inline README.md:198]:
```suggestion
	"cardBin":"string",
```

Tabs vs. spaces 🤷‍♂️

**Review by ct-clearhaus — COMMENTED** [review]
I notice from the examples that you're returning "BIN8" — so in fact you're only truncating 4, not 6 digits of a 16 digit PAN/CAN (because `maskedCardNumber` has "last4" and `cardBin` has "BIN8") — did you give this thoughts in relation to PCI DSS?

File: README.md (hunk 1 above)
Comment (mniss) [reply README.md]: @ct-clearhaus: You're correct. In effect, we'll be returning 8+4 digits (when combining maskedCardNumber + cardBin). This has been assessed internally to comply with PCI-DSS

Comment (ct-clearhaus) [reply README.md]: @mniss, thanks a bunch! Maybe I'm slow, but can you ELI5 me how this conforms to PCI DSS Req. 3.3:

<kbd>
Screenshot 2022-05-03 at 09 22 57
</kbd>

Maybe you're expecting the recipient to get the response in a CDE?

Comment (mniss) [reply README.md]: @ct-clearhaus , I've sent you an e-mail :)
